x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/parser/AbstractBuilder.java

```diff
@@ -173,6 +177,13 @@ public static String unquoteString(String text) {
         return resultString.toString();
     }
 
+    private static void checkForSingleQuotedString(Source source, String text, int i) {
+        if (text.charAt(i) == '\'') {
+            throw new ParsingException(source,
+                    "Single quotes ['] are not supported around literal strings, please use double quotes [\"]");
```

Comment:
`Use double quotes ["] for defining string literals, not single quotes [']`

Reply:
I will make it: `Use double quotes ["] to define string literals, not single quotes [']`